backend/src/test/java/codezap/template/service/TemplateServiceTest.java

```diff
@@ -7,272 +7,566 @@
-        );
-    }
+        @Test
+        @DisplayName("멤버 id로 템플릿 조회 성공")
+        void getByMemberIdSuccess() {
+            var member1 = memberRepository.save(MemberFixture.getFirstMember());
+            var member2 = memberRepository.save(MemberFixture.getSecondMember());
+            var category = categoryRepository.save(CategoryFixture.getFirstCategory());
+            var template1 = templateRepository.save(TemplateFixture.get(member1, category));
+            var template2 = templateRepository.save(TemplateFixture.get(member1, category));
+            var template3 = templateRepository.save(TemplateFixture.get(member2, category));
 
-    @Test
-    @DisplayName("템플릿 수정 성공")
-    void updateTemplateSuccess() {
-        // given
-        Member member = MemberFixture.getFirstMember();
-        Category category = CategoryFixture.getFirstCategory();
-        CreateTemplateRequest createdTemplate = makeTemplateRequest("title");
-        Template template = saveTemplate(createdTemplate, category, member);
-        categoryRepository.save(category);
-
-        // when
-        UpdateTemplateRequest updateTemplateRequest = makeUpdateTemplateRequest(1L);
-        templateService.updateTemplate(member, template.getId(), updateTemplateRequest, category);
-        Template updateTemplate = templateRepository.fetchById(template.getId());
-        List<Tag> tags = templateTagRepository.findAllByTemplate(updateTemplate).stream()
-                .map(TemplateTag::getTag)
-                .toList();
-
-        // then
-        assertAll(
-                () -> assertThat(updateTemplate.getTitle()).isEqualTo("updateTitle"),
-                () -> assertThat(updateTemplate.getCategory().getId()).isEqualTo(1L),
-                () -> assertThat(tags).hasSize(2)
-        );
-    }
+            var actual = sut.getByMemberId(member1.getId());
 
-    @Test
-    @DisplayName("템플릿 수정 실패: 권한 없음")
-    void updateTemplateFailWithUnauthorized() {
-        // given
-        Member member = MemberFixture.getFirstMember();
-        Category category = CategoryFixture.getFirstCategory();
-        CreateTemplateRequest createdTemplate = makeTemplateRequest("title");
-        Template template = saveTemplate(createdTemplate, new Category("category1", member), member);
-        categoryRepository.save(new Category("category2", member));
-
-        // when
-        Member otherMember = MemberFixture.getSecondMember();
-        UpdateTemplateRequest updateTemplateRequest = makeUpdateTemplateRequest(2L);
-
-        // then
-        Long templateId = template.getId();
-        assertThatThrownBy(
-                () -> templateService.updateTemplate(otherMember, templateId, updateTemplateRequest, category))
-                .isInstanceOf(CodeZapException.class)
-                .hasMessage("해당 템플릿에 대한 권한이 없습니다.");
-    }
+            assertThat(actual).hasSize(2)
+                    .containsExactly(template1, template2);
```

Comment:
다른 멤버꺼는 보여지지 않는다는 의미로 doesNotContain까지 해주는 건 어떤가요?
근데 생각해보면 containsExactly로 정확한 비교를 하니 없어도 될 것 같기도 ~!

```suggestion
            assertThat(actual).hasSize(2)
                    .containsExactly(template1, template2);
                     .doesNotContain(FindCategoryResponse.from(template3));
```

Reply:
더욱 명시적으로 나타낼 수는 있다고 생각하지만 켬미가 얘기한 것처럼 `containsExactly`를 사용하기 때문에 불필요하다고 생각됩니다